azure-simple/packer/ansible-is/README.md

```diff
@@ -0,0 +1,126 @@
+│   │   ├── Deployment-pattern-1-diagram.png
+│   │   └── Deployment-pattern-2-diagram.png
+│   ├── Pattern1.md
+│   └── Pattern2.md
+├── files
+│   ├── lib
+│   │   ├── amazon-corretto-8.222.10.1-linux-x64.tar.gz
+│   │   └── mysql-connector-java-5.1.47-bin.jar
+│   └── packs
+│       └── wso2is-5.10.0.zip
+├── issue_template.md
+├── LICENSE
+├── pull_request_template.md
+├── README.md
+├── roles
+│   ├── common
+│   │   └── tasks
+│   │       ├── custom.yml
+│   │       └── main.yml
+│   └── is
+│       ├── tasks
+│       │   ├── custom.yml
+│       │   └── main.yml
+│       └── templates
+│           ├── carbon-home
+│           │   ├── bin
+│           │   │   └── wso2server.sh.j2
+│           │   └── repository
+│           │       └── conf
+│           │           └── deployment.toml.j2
+│           └── wso2is.service.j2
+├── scripts
+│   ├── update.sh
+│   └── update_README.md
+└── site.yml
+```
+
+Packs could be either copied to a local directory, or downloaded from a remote location.
+
+## Packs to be Copied
+
+Copy the following files to `files/packs` directory.
+
+1. [WSO2 Identity Server 5.10.0 package](https://wso2.com/identity-and-access-management/install)
+
+Copy the following files to `files/lib` directory.
+
+1. [MySQL Connector/J](https://dev.mysql.com/downloads/connector/j/5.1.html)
+2. [Amazon Coretto for Linux x64 JDK](https://docs.aws.amazon.com/corretto/latest/corretto-8-ug/downloads-list.html)
+
+## Downloading from remote location
+
+In **group_vars**, change the values of the following variables in all groups:
+1. The value of `pack_location` should be changed from "local" to "remote"
+2. The value of `remote_jdk` should be changed to the URL in which the JDK should be downloaded from, and remove it as a comment.
+3. The value of `remote_pack` should be changed to the URL in which the package should be downloaded from, and remove it as a comment.
+
+## Running WSO2 Identity Server Ansible scripts
+
+### 1. Run the existing scripts without customization
+The existing Ansible scripts contain the configurations to set-up a single node WSO2 Identity Server pattern. In order to deploy the pattern, you need to replace the `[ip_address]` given in the `inventory` file under `dev` folder by the IP of the location where you need to host the Identity Server. An example is given below.
+```
+[is]
+wso2is ansible_host=172.28.128.4
+```
+
+Run the following command to run the scripts.
+
+`ansible-playbook -i dev site.yml`
+
+If you need to alter the configurations given, please change the parameterized values in the yaml files under `group_vars` and `host_vars`.
+
+### 2. Customize the WSO2 Ansible scripts
+
+The templates that are used by the Ansible scripts are in j2 format in-order to enable parameterization.
+
+The `axis2.xml.j2` file is added under `roles/is/templates/carbon-home/repositoy/conf/axis2/`, in order to enable customizations. You can add any other customizations to `custom.yml` under tasks of each role as well.
```

Comment:
@rasanjaya85 shouldn't these examples actually point to `deployment.toml` file from now on?

If I remember the WSO2 recommended best practices, we actually don't meddle with other files, unless it is absolutely necessary. Please correct me if I am wrong.